packages/core/src/components/Card/story.tsx

```diff
@@ -332,3 +332,61 @@ export function cardAsAButtonWithMiddleAlignment() {
 cardAsAButtonWithMiddleAlignment.story = {
   name: 'Card as a button with middle alignment.',
 };
+
+export function asFullSizeCustomBefore() {
+  return (
+    <Card>
+      <Content
+        beforeFullSize
+        before={
+          <ResponsiveImage
+            noShadow
+            alt="Image"
+            borderRadius={3}
+            maxHeight={300}
+            maxWidth={300}
+            src={stars}
+          />
+        }
+      >
+        <Text truncated>
+          <LoremIpsum />
+        </Text>
+      </Content>
+    </Card>
+  );
+}
+
+asFullSizeCustomBefore.story = {
+  name: 'A card with full size before content.',
+};
+
+export function asFullSizeCustomAfter() {
+  return (
+    <Card>
+      <Content
+        truncated
+        afterFullSize
+        after={
+          <ResponsiveImage
```

Comment:
this and the above examples of `afterFullSize` + `after` is doing what `afterImageSrc` does. if the goal is to make the before/after images clickable, add `onClickBeforeImage` and `onClickAfterImage` to keep before/after image treatments consistent. before/after content is supposed to have the padding.

Reply:
It seems like enforcing style here might not be a scalable approach.  When we could make the component more flexible and less complex by removing before/after image src, as well as the large/small props, and just allowing you to define your own padding.  I can certainly just start adding more props on to do what we want, but it feels like it will just continue to bloat the component

@stefhatcher

Reply:
yeah, but then we'd lose consistency with images in cards in the design system.

Reply:
We should just redo card for v3. It's become to unmanageable.

Reply:
I would agree with that.  Short term both of you agree I should go and add click handler?

Reply:
If so, any recommendation on defining the prop type there?  I am not super familiar with airbnb prop types library, and it feels complex since it's mutually exlcusive with before/after, but also requires beforeImageSrc/afterImageSrc

Reply:
since we'll do a re-work, maybe it's fine to just do the check `onClickBeforeImageSrc && beforeImageSrc` in the code, no extra prop type def besides the callbacks being functions